ServiceProvider/MakerServiceProvider.php

```diff
@@ -33,68 +47,74 @@ public function register(BaseApplication $app)
             return $app['orm.em']->getRepository('Plugin\Maker\Entity\ProductMaker');
         });
 
+        // Maker event
+        $app['eccube.plugin.maker.event.maker'] = $app->share(function () use ($app) {
+            return new Maker($app);
+        });
+
+        // Maker legacy event
+        $app['eccube.plugin.maker.event.maker_legacy'] = $app->share(function () use ($app) {
+            return new MakerLegacy($app);
+        });
+
         // 一覧・登録・修正
-        $app->match('/' . $app["config"]["admin_route"] . '/product/maker/{id}', '\\Plugin\\Maker\\Controller\\MakerController::index')
+        $app->match('/'.$app['config']['admin_route'].'/plugin/maker/{id}', '\\Plugin\\Maker\\Controller\\MakerController::index')
```

Comment:
Don't add routing directory via `$app`. Use `$admin->match` instead of `$app->match` for force SSL support.
(`$app`を使って直接ルーティングを追加するのではなく、強制SSLの対応のために`$admin`を使ってください。)

```
// 管理画面定義
$admin = $app['controllers_factory'];
// 強制SSL
if ($app['config']['force_ssl'] == Constant::ENABLED) {
    $admin->requireHttps();
}
// 一覧・登録・修正
$admin->match('/plugin/maker/{id}', '\\Plugin\\Maker\\Controller\\MakerController::index')
    ->value('id', null)->assert('id', '\d+|')
    ->bind('admin_plugin_maker_index');

...

$app->mount('/'.trim($app['config']['admin_route'], '/').'/', $admin);
```

See https://github.com/k-yamamura/ec-cube.github.io/blob/feature-plugin-bestpractices/plugin/controller.md#%E3%82%B3%E3%83%B3%E3%83%88%E3%83%AD%E3%83%BC%E3%83%A9%E3%83%BC%E3%81%AE%E3%83%AB%E3%83%BC%E3%83%86%E3%82%A3%E3%83%B3%E3%82%B0%E5%AE%9A%E7%BE%A9